tests/integration/devfile/cmd_devfile_catalog_test.go

```diff
@@ -22,7 +23,12 @@ var _ = Describe("odo devfile catalog command tests", func() {
 		os.Setenv("GLOBALODOCONFIG", filepath.Join(context, "config.yaml"))
 		helper.CmdShouldPass("odo", "preference", "set", "Experimental", "true")
 		if os.Getenv("KUBERNETES") == "true" {
-			project = helper.CreateRandNamespace(context)
+			err := helper.CopyFile(os.Getenv("KUBECONFIG"), filepath.Join(context, "config"))
+			if err != nil {
+				fmt.Println("Source file does not copied to the target with error", err)
```

Comment:
This need to be asserted instead of just printing. For example - https://github.com/openshift/odo/blob/master/tests/helper/helper_filesystem.go#L92.